packages/flutter_tools/templates/plugin/test/projectName_test.dart.tmpl

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:{{projectName}}/{{projectName}}.dart';
+import 'package:mockito/mockito.dart';
+
+void main() {
+  MockMethodChannel methodChannel;
+  {{pluginDartClass}} {{projectName}};
+
+  setUp(() {
+    methodChannel = MockMethodChannel();
+    {{projectName}} = {{pluginDartClass}}.private(methodChannel);
+  });
+
+  test('getPlatformVersion', () async {
+    when(methodChannel.invokeMethod('getPlatformVersion'))
+        .thenAnswer((Invocation invoke) => Future<String>.value('42'));
+    expect(await {{projectName}}.platformVersion, '42');
+  });
+}
+
+class MockMethodChannel extends Mock implements MethodChannel {}
```

Comment:
```suggestion
class MockMethodChannel extends Mock implements MethodChannel {}

```

Reply:
is this neccessary? this file already has empty line at the end, if we add one more line as this suggestion, we end up with 2 empty lines.

Reply:
My mistake!